bin/citgm.js

```diff
@@ -84,8 +85,10 @@ function launch(mod, options) {
   }).on('data', function(type, key, message) {
     log[type](key, message);
   }).on('end', function(module) {
+    module.duration = new Date() - start;
     reporter.logger(log, module);
 
+    log.info('duration', 'test time: ' + module.duration + 'ms');
```

Comment:
nit: this should be `test duration:`

will fix this on landing if it is the only issue